docs/en/Remote-Logging.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: global
+title: Remote Logging
+group: Features
+priority: 5
+---
+
+* Table of Contents
+{:toc}
+
+## Overview
+Alluxio supports sending logs to a remote log server over the network. This feature can be useful
+to system administrators who have to perform the task of log collection. With remote logging, the
+log files, e.g. master.log, worker.log, etc. on all Alluxio servers will be readily available on
+a designated and configurable directory on the log server.
+
+## Configure Alluxio
+Suppose the hostname of the log server is AlluxioLogServer, and the port is 45010.
```

Comment:
Define the hostname and log server port closer to where they are used, or let the context explain what they mean

Reply:
Done.